platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -247,7 +247,7 @@ NS_ASSUME_NONNULL_BEGIN
  @param annotation The annotation being rendered.
  @return A line width for the polyline, measured in points.
  */
-- (CGFloat)mapView:(MGLMapView *)mapView lineWidthForPolylineAnnotation:(MGLPolyline *)annotation;
+- (float)mapView:(MGLMapView *)mapView lineWidthForPolylineAnnotation:(MGLPolyline *)annotation;
```

Comment:
This would be a backwards-incompatible change on 64-bit systems, where `CGFloat` is a typedef for `double`. Values as large as `FLT_MAX` are nonsensical, but it’s a convention in Cocoa APIs to use `CGFloat` for any value measured in screen coordinates. This should be reverted to `CGFloat` for consistency with all the other geometry APIs in this SDK and in Cocoa frameworks.